distribution/test/itests/test-itests-alliance/src/test/java/org/codice/alliance/test/itests/SecurityAuditPluginTest.java

```diff
@@ -0,0 +1,97 @@
+/**
+ * Copyright (c) Codice Foundation
+ * <p>
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser
+ * General Public License as published by the Free Software Foundation, either version 3 of the
+ * License, or any later version.
+ * <p>
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without
+ * even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details. A copy of the GNU Lesser General Public License
+ * is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+package org.codice.alliance.test.itests;
+
+import static org.codice.ddf.itests.common.catalog.CatalogTestCommons.deleteMetacard;
+import static org.codice.ddf.itests.common.catalog.CatalogTestCommons.ingest;
+import static org.codice.ddf.itests.common.catalog.CatalogTestCommons.update;
+import static org.junit.Assert.assertTrue;
+
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.ArrayList;
+import java.util.Dictionary;
+import java.util.Hashtable;
+import java.util.List;
+
+import org.apache.commons.io.IOUtils;
+import org.codice.alliance.test.itests.common.AbstractAllianceIntegrationTest;
+import org.codice.ddf.itests.common.annotations.BeforeExam;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.ops4j.pax.exam.junit.PaxExam;
+import org.ops4j.pax.exam.spi.reactors.ExamReactorStrategy;
+import org.ops4j.pax.exam.spi.reactors.PerClass;
+import org.osgi.service.cm.Configuration;
+
+@RunWith(PaxExam.class)
+@ExamReactorStrategy(PerClass.class)
+public class SecurityAuditPluginTest extends AbstractAllianceIntegrationTest {
+    private static final String[] REQUIRED_APPS =
+            {"catalog-app", "solr-app", "spatial-app", "security-app"};
+
+    private String auditMessageFormat =
+            "Attribute %s on metacard %s with value(s) %s was updated to value(s) %s";
+
+    @BeforeExam
+    public void beforeExam() throws Exception {
+        basePort = getBasePort();
+        getAdminConfig().setLogLevels();
+
+        getServiceManager().waitForRequiredApps(REQUIRED_APPS);
+        getServiceManager().waitForAllBundles();
+        getCatalogBundle().waitForCatalogProvider();
+
+    }
+
+    @Test
+    public void testSecurityAuditPlugin() throws Exception {
+        Configuration config = configAdmin.getConfiguration(
+                "org.codice.alliance.catalog.plugin.security.audit.SecurityAuditPlugin",
+                null);
+        List attributes = new ArrayList<>();
+        attributes.add("description");
+        Dictionary properties = new Hashtable<>();
+        properties.put("auditAttributes", attributes);
+        config.update(properties);
+        getServiceManager().waitForAllBundles();
```

Comment:
We shouldn't have to call `waitForAllBundles()` here. Instead, we should wait for the log message that says that the configuration has been updated and then move on.